RealLifeGTAV/dist/metadata.json

```diff
@@ -4,31 +4,22 @@
     "name": "N0chteil ^^",
     "id": "495901098926669825"
   },
-  "contributors": [
-    {
-      "name": "N0chteil",
-      "id": "495901098926669825"
-    }
-  ],
-  "url": [
-    "reallifegtav.de",
-    "www.reallifegtav.de"
-  ],
+  "service": "RealLifeGTAV",
   "description": {
     "en": "Real Life GTAV is a german FiveM roleplay server with custom scripts and more!",
-	  "de": "Real Life GTAV ist ein deutscher FiveM Roleplay Server mit eigenen Scripts und mehr!"
+    "de": "Real Life GTAV ist ein deutscher FiveM Roleplay Server mit eigenen Scripts und mehr!"
   },
-  "service": "Real Life GTAV",
+  "url": [
+    "reallifegtav.de"
+  ],
```

Comment:
doesnt need to be in an array if its only 1

Reply:
fixed, thanks.